### ISSUE TITLE
logical defs for beta-amylase activity have gone strange

### ISSUE THREAD

**Issue by cmungall** [Planteome/plant-trait-ontology#394]
we used to have:

```
[Term]
id: TO:0000338
name: beta-amylase activity
intersection_of: PATO:0001236 ! process quality
intersection_of: inheres_in GO:0016161 ! beta-amylase activity
```

which is reasonable. However, this has changed to

```
id: TO:0000338
name: beta-amylase activity
intersection_of: PATO:0000001 ! quality
intersection_of: inheres_in TO:0000338 ! beta-amylase activity
```

so we now have a circular def. The release is failing:

```
Self Reference In Definition	Error	

		Class TO:0000338 has a self reference in its logical definition: EquivalentClasses(<http://purl.obolibrary.org/obo/TO_0000338> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PATO_0000001> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000052> <http://purl.obolibrary.org/obo/TO_0000338>)) )

		Class TO:0000288 has a self reference in its logical definition: EquivalentClasses(<http://purl.obolibrary.org/obo/TO_0000288> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PATO_0000001> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000052> <http://purl.obolibrary.org/obo/TO_0000288>)) )

		Class TO:0000705 has a self reference in its logical definition: EquivalentClasses(<http://purl.obolibrary.org/obo/TO_0000705> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PATO_0000001> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000052> <http://purl.obolibrary.org/obo/TO_0000705>)) )
```

**Comment by cmungall** [comment]
Note that Protege can get confused when classes have the same label.

A TO class should __never__ have the same label as a GO class. These 3 traits should be named as traits, not activities. Suffixing 'trait' onto the label should be enough

**Comment by cooperl09** [comment]
Fixed in commit b654d305f09d44762b82596cf7221eee7c5f8902
